client/verta/verta/data_types/_numeric_value.py

```diff
@@ -60,6 +60,27 @@ def _from_dict_inner(cls, d):
         )
 
     def dist(self, other):
+        r"""Calculate the distance from `other` to this value.
+
+        Parameters
+        ----------
+        other : :class:`NumericValue`
+            Value to calculate distance from.
+
+        Returns
+        -------
+        float
+            Relative difference between the values.
+
+        Notes
+        -----
+        This method uses relative difference:
+
+        .. math::
+
+            \text{dist}=\left\lvert\frac{\text{self}-\text{other}}{\text{other}}\right\rvert
```

Comment:
![Screen Shot 2021-06-03 at 9 31 23 AM](https://user-images.githubusercontent.com/7754936/120680006-acc4bd80-c44e-11eb-9851-e71f165260a6.png)

Reply:
☝️ dist has been changed to diff